stl/inc/yvals_core.h

```diff
@@ -447,6 +454,12 @@
 #define _MSVC_KNOWN_SEMANTICS
 #endif
 
+#define _MSVC_NOOP_DTOR [[msvc::noop_dtor]]

```

Comment:
Pre-existing, but I suspect that we should use the same logic as for `msvc::known_semantics` immediately above. (They could be partially fused, but replicating is probably easier to understand/maintain.)

Reply:
I considered this, and decided to leave well enough alone since it was working. But I can certainly imagine it becoming a problem someday, so I don't mind making the change.

Reply:
Instead of fusing, I factored out the control into a (local temporary) function-like macro. I think this is as readable as "duplicate everything", but will scale better if we add more such attributes in the future.